### PR TITLE
Auto create story distros

### PR DESCRIPTION
**Pull request by kookster**
- [x] create story distributions when a story is created and is in a series that has distributions associated with it
- [x] send story attributes to feeder on episode create
- [x] include distributions in the story representer
- [x] bug fix: don't include stories in the series min representer - makes it huge

### CODE REVIEW

File: app/representers/api/story_representer.rb [hunk 1]
```diff
@@ -110,6 +110,17 @@ class Api::StoryRepresenter < Api::BaseRepresenter
     } if represented.id
   end
   embed :musical_works, paged: true, item_class: MusicalWork, zoom: false
+
+  link :distributions do
+    {
```
Comment (kookster) [inline app/representers/api/story_representer.rb]: no, I like it how it is

File: test/models/story_test.rb [hunk 2]
```diff
@@ -26,6 +26,29 @@
...
+    it 'creates story distributions' do
+      stub_request(:post, 'https://id.prx.org/token').
+        to_return(status: 200,
+                  body: '{"access_token":"abc123","token_type":"bearer"}',
+                  headers: { 'Content-Type' => 'application/json; charset=utf-8' })
+
+      stub_request(:get, 'https://feeder.prx.org/api/v1/podcasts/23').
+        to_return(status: 200, body: json_file(:podcast), headers: {})
+
+      stub_request(:post, 'https://feeder.prx.org/api/v1/podcasts/23/episodes').
```
Comment (cavis) [inline test/models/story_test.rb]: Does webmock automatically assert that a stubbed request was actually made?

File: app/models/story.rb [hunk 3]
```diff
@@ -212,6 +214,19 @@ def v4?
...
 
+  # for each distribution, create a story distribution
+  def create_story_distributions
+    return unless series
+    series.distributions.each do |distro|
+      begin
+        distro.create_story_distribution(self)
+      rescue StandardError => err
+        logger.error(err)
+        NewRelic::Agent.notice_error(err)
```
Comment (cavis) [inline app/models/story.rb]: I like this.  Any case for also adding this to the series, when it's creating the feeder Podcast?

File: test/models/story_test.rb (hunk 2 above)
Comment (kookster) [inline test/models/story_test.rb]: you could say that - it throws an error so the test errors out if you don't stub the request.

File: app/models/story.rb (hunk 3 above)
Comment (kookster) [inline app/models/story.rb]: yes, we can expand using this - copying what you did in dovetail. I think if we want to do this more in the rails apps, we should probably abstract it, on the off chance that we quit using new relic, or so we can easily stub or change all error noticing

**Pull request merged**